web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -1512,12 +1512,23 @@ $.extend(fixmystreet.set_up, {
     });
     $('body').on('click', '#alert_email_button', function(e) {
         e.preventDefault();
+
+        var emailInput = $(this).closest('.js-alert-list').find('input[type=email]');
+        emailInput[0].required = true;
```

Comment:
I missed this previously, but if you're logged in, there is no emailInput here (because we don't always show an email box then) and so this line errors. Am fixing now, sorry for missing.